tests/Neo.Plugins.OracleService.Tests/config.json

```diff
@@ -39,29 +39,9 @@
       "HF_Basilisk": 4120000
     },
     "InitialGasDistribution": 5200000000000000,
-    "ValidatorsCount": 7,
+    "ValidatorsCount": 1,
```

Comment:
Why reduce the validators for testing?

Reply:
> Why reduce the validators for testing?

Because the original oracle tests does not include any 7-consensus steps. The old config was simply a copy of mainnet.